### PR TITLE
feat: 🎸 Stepper Component for Angular

### PR DESCRIPTION
**Pull request by NewtonJoshua**
Stepper Component for Angular

Stepper Component for the Wizard
https://momentum.design/components/stepper/style

## Description
https://momentum.design/components/stepper/style
https://momentum.design/guidelines/wizards

## Motivation and Context
This is the implementation of the guidelines for Wizard

## How Has This Been Tested?
This is tested using the examples component

## Screenshots:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .angulardoc.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "repoId": "4c12df35-5086-47f0-ab90-e3e021deea04",
+  "lastSync": 0
+}
```
Comment (JohnChambersBestFriend) [inline .angulardoc.json]: [remove] - this is a VS Code config file that should not be added to the project.

File: angular/src/docs/app/app-routing.module.ts [hunk 2]
```diff
@@ -27,6 +28,10 @@ const routes: Routes = [
         path: 'radio',
         component: ExampleRadioKitchenSinkComponent,
       },
+      {
+        path: 'stepper',
+        component: StepperDefaultComponent,
```
Comment (bfbiggs) [inline angular/src/docs/app/app-routing.module.ts]: Follow format of ExampleStepperKitchenSinkComponent

File: angular/src/lib/stepper/stepper.component.scss [hunk 3]
```diff
@@ -0,0 +1,129 @@
+.md-stepper-container {
```
Comment (bfbiggs) [inline angular/src/lib/stepper/stepper.component.scss]: This scss should be moved to core directory scss file.

File: angular/src/lib/stepper/stepper.component.scss [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  display: inline-block;
+  vertical-align: top;
+  text-align: center;
+  margin: 2%;
+}
+
+.md-stepper-bubble {
+  border-radius: 50%;
+  display: inline-block;
+  background: #FFFFFF;
```
Comment (bfbiggs) [inline angular/src/lib/stepper/stepper.component.scss]: Use momentum color variables if possible.

File: angular/src/lib/stepper/stepper.component.ts [hunk 5]
```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit, Input, Output, EventEmitter, AfterViewInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'md-stepper',
+  templateUrl: './stepper.component.html',
+  styleUrls: ['./stepper.component.scss'],
+})
+export class StepperComponent implements OnInit, AfterViewInit {
+  @Input() public steps: {
```
Comment (bfbiggs) [inline angular/src/lib/stepper/stepper.component.ts]: Missing @prop notations.  @daanishnasir can help clarify the style needed.

File: .angulardoc.json (hunk 1 above)
Comment (NewtonJoshua) [reply .angulardoc.json]: Done

File: angular/src/docs/app/app-routing.module.ts (hunk 2 above)
Comment (NewtonJoshua) [reply angular/src/docs/app/app-routing.module.ts]: Done

File: angular/src/lib/stepper/stepper.component.scss (hunk 3 above)
Comment (NewtonJoshua) [reply angular/src/lib/stepper/stepper.component.scss]: Done

File: angular/src/lib/stepper/stepper.component.scss (hunk 4 above)
Comment (NewtonJoshua) [reply angular/src/lib/stepper/stepper.component.scss]: Done

File: angular/src/lib/stepper/stepper.component.ts (hunk 5 above)
Comment (NewtonJoshua) [reply angular/src/lib/stepper/stepper.component.ts]: Done

**Comment by wzpo** [comment]
@NewtonJoshua Please fix the conflicts, so we can merge it to the master.

**Comment by NewtonJoshua** [comment]
@wzpo , @ArthusLiang : 
Please note that the branch is updated and there are no conflicts.
Please proceed with merging the branch

**Pull request closed without merging**